LICENSE.note

```diff
@@ -0,0 +1,81 @@
+vdiffr as a whole is released under MIT. It includes:
+
+- the jQuery library under the Apache 2.0 licence,
+- the js-imagediff library under the MIT licence,
+- the TwoFace library under the MIT licence,
+- the daff library under the MIT license,
+- the diff2html library under the MIT license,
+
+all of which are compatible with the MIT.
+
+
+js-imagediff
```

Comment:
I don't think we need to include the source of the licenses here. They can continue to live in the respective subdirs.